test/missing_tests.jl

```diff
@@ -15,23 +15,24 @@ end
     @test MCMCChains.cummean(x) == [1., 1., 2., 2.]
 end
 
-@testset "stats" begin
-    Random.seed!(1234)
-    chn = Chains(randn(1000, 2, 2))
+# for now, summarystats will return missing whenever the chain contains one or more missing values
+# @testset "stats" begin
+#     Random.seed!(1234)
+#     chn = Chains(randn(1000, 2, 2))
 
-    # Call describe without missing values.
-    describe(devnull, chn; sections = nothing)
-    s1, s2 = summarystats(chn), quantile(chn)
+#     # Call describe without missing values.
+#     describe(devnull, chn; sections = nothing)
+#     s1, s2 = summarystats(chn), quantile(chn)
 
-    # Add missing values.
-    chn_m = Chains(cat(chn.value, ones(1, 2, 2) .* missing, dims = 1))
+#     # Add missing values.
+#     chn_m = Chains(cat(chn.value, ones(1, 2, 2) .* missing, dims = 1))
 
-    # Call describe with missing values.
-    describe(devnull, chn_m; sections = nothing)
-    m1, m2 = summarystats(chn_m), quantile(chn_m)
+#     # Call describe with missing values.
+#     describe(devnull, chn_m; sections = nothing)
+#     m1, m2 = summarystats(chn_m), quantile(chn_m)
 
-    @test testdiff(s1, m1)
-end
+#     @test testdiff(s1, m1)
+# end
```

Comment:
In general, I think it is better to remove stuff than to comment it (it will still be in the git history if we agree to remove it).